src/cli/onefuzz/api.py

```diff
@@ -153,6 +158,42 @@ def _req_model_list(
 
         return [model.parse_obj(x) for x in response]
 
+    def _req_config_params(
+        self,
+    ) -> None:
+
+        endpoint_params = responses.Config(
+            authority="",
+            client_id="",
+            tenant_domain="",
+        )
+
+        if self.onefuzz._backend.config.endpoint is not None:
```

Comment:
Just a stylistic thing but I'd invert the if to do the check and throw first, to reduce overall indentation

Reply:
Done.